README.md

```diff
@@ -1,3 +1,13 @@
 # IBPSA Project 1 - District Energy Simulation Test Procedure (DESTEST)
 
-This repository contains the DESTEST developed within the IBPSA Project 1.
+To establish reliable, decarbonized and sustainable cities, it is necessary to integrate all energy grids together with the different energy suppliers and end-users during the design, optimization and operation phases. To that matter, accurate urban-scale numerical models have to be developed to dynamically simulate buildings, infrastructures, networks, energy and matter fluxes. The multi-physics modelling language Modelica is particularly well-suited for that purpose.
+
+The goal of the IBPSA Project 1 is to develop a BIM/GIS and Modelica framework for building and community energy simulation. Within the IBPSA Project 1, the DESTEST (Work Package 3.1) aims at testing district energy system simulations tools, and in particular, validate the urban energy systems models of the dedicated Modelica libraries.
```

Comment:
Here I would add the links to the libraries and add a sentence like: The dedicated Modelica libraries can be found here: and they are based on the modelica-ibpsa (link) core library.